doc/examples/142-https-client-cert/config.json

```diff
@@ -3,7 +3,7 @@
NVBAcMDERlZmF1bHQgQ2l0eTEcMBoGA1UE\nCgwTRGVmYXVsdCBDb21wYW55IEx0ZDEsMCoGA1UEAwwjbW9jay1zZXJ2ZXItaHR0\ncHMtY2xpZW50LWNlcnQtcHJveHkwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEK\nAoIBAQCz5kYVCP4wrE82qqucixz0KcigaUjDkM9pYlGlR9DLmzkkvoLWqqNTnr8E\ncNxidIo41xdkc+RyojxOCEwaaeIWq1J1V3is/ZSsv6JN6Kz43kDAjU0v7AjBgwbg\n3o+ri+lRJPpoP/HEAuC3Yl0ATr7asM6SuwJ68FaN4xcd7hbLN+KnqPHfH/wOuVR0\nWjXkPoK/L+fRG3wwyv5j6ta92d6xc/Y9fYDtzNY1Za3OgKOUs+jkdhg7nWhBx4SE\nh3hsLhWQSMIh5tKulfagCPpFQsx/vZ07rbb/A/B7ep9apD1y6T1/y4FwQ7DTpP/P\n5XWH6odAtlYsLi6kOx2eJAomF/7xAgMBAAEwDQYJKoZIhvcNAQELBQADggIBAG15\nohjpZfEpKhSvdr/CtwG6uLmeIX9y2DhSNKVmPWL1R2zW3MG3K0peWAkK/wYOmEHV\ngUkNI4aLWydU6Y36itTfBAgtbCwUJmUyqen78mXyFV6hloQoX9l8naeC0/LBygHV\nkAeO5i+kWBeI6LZ/MBzlUSt4g+D0RyKQiQIRtU2jzLXClsn3k7q1cPzYy/8XE5Zd\nnvxjmMwQYTgy6yVZTTgiBzvPi7QiA0GZ4fU4FWsU73C1KliqKV1m43v6KvP+1j2l\nDAzvt0buP78vkyucUxtCBCvyGXXcrrN4S9rkiGdr1RZ2Si6vzI0q0AODLzYQr5kW\nIN+jNtYD4y99+W2vHLNGDvQyAe/ePG0NWOeJm72qQmb76610hEomJxuH1ZmpnD/f\ncj8YPcKoW9eJ3i3WCI6MwEWDLf1bvAnFXTchRlVkNiCZ/SdQwdTcQzDFCi7qrBpP\nML/tnAdBpWRxjdtjn4ZO2B6Qc638hpljEko5YKkKeHCNFAYeCFsmj6/KOhhBSV8/\nqY/0MjTRSyE/o9AL+3rANXuWdXnzoNFObwRcbCCGjb/a7of/YDhVgYRImZqiLGCM\nTuJLTaYLrUWIIny0dcRCBD84xb7Os488QuuVrSwWC1nRaqeWsKS53nNsETjobmWZ\nLZ3qWxVhDp+DcnJefjBpWvK2t7YhBLWmKI6W4D1Y\n-----END CERTIFICATE-----\n-----BEGIN RSA PRIVATE KEY-----\nMIIEpAIBAAKCAQEAs+ZGFQj+MKxPNqqrnIsc9CnIoGlIw5DPaWJRpUfQy5s5JL6C\n1qqjU56/BHDcYnSKONcXZHPkcqI8TghMGmniFqtSdVd4rP2UrL+iTeis+N5AwI1N\nL+wIwYMG4N6Pq4vpUST6aD/xxALgt2JdAE6+2rDOkrsCevBWjeMXHe4Wyzfip6jx\n3x/8DrlUdFo15D6Cvy/n0Rt8MMr+Y+rWvdnesXP2PX2A7czWNWWtzoCjlLPo5HYY\nO51oQceEhId4bC4VkEjCIebSrpX2oAj6RULMf72dO622/wPwe3qfWqQ9cuk9f8uB\ncEOw06T/z+V1h+qHQLZWLC4upDsdniQKJhf+8QIDAQABAoIBAQCoWTiCHM1KnSTv\nwl0f/GrW7ncJ7hyirlP6iYBgDEmPD640/XcRDdQTv31/2WHTXL2wnVANuAozYDDQ\n0SDuouFpV1+t4XJTaGYGFrr6719W8R1lU5XE37v94vC/5GZMIXzwa4tNnXFQfWTv\nF8RyMj6WBlj5pLmYdGp7EuUiJBj+h7bS+VloKc8ZujrYm02DPUV0u3FjoXRvf4mo\nETSkO085xBIGHsrY3OnSHLku8MuklbeqA/1eyJQR6+eXIp3gS5tN3xV59LmtXuZu\n4VK6YDRqurRaLFekxnnVLQ//E8yxqstpY9SHPm0m2UYdEoq8zx8lNt7m2CkhpjU/\n8OPQMmI1AoGBAN5QP/AhHmYzwUmEnVGd5KqG8ZmmzmeNhZxIujrf7J+kkgYMROKs\nTtoD4hcE94tBBi9dhHb55/UiwE9z40awuH42UuDhl+HoggLrJQLvHC0Vke1kckQx\n/GgYfRDbQykeOrF8GFalq0KEB9tOLmwck40HNVI0CCQSI5nPOBGFaYubAoGBAM8o\nvvkYZMJgDcaCZeWDJSApg1VtlOXIUTZYcEiPduBZiVdwxSOMTrWZXsaDKrSVJCZu\n7lwh+xjSmTgMpVNPWXm1VzIn0UPXD3UJeTZhfyRPK97FmgUit5GwMnFwuXPY5MYT\nsLRoB7FY/dNlbpcMeAzE7CVktg0zguA8thEGiiZjAoGAMh91yvYsnG5ezeyQa65l\nt+zk2/e34bEOxsW+9zc1amj4Y1Sw00p2HMFzYC1GgqcOde9nxkth+Or04PUH8RA3\nCH3HAdzbGZwKdA4AMkEm4SNT22oiOS3r8qxSxlIpOkQOh6Wq9x4zFrQd8VwXK1jm\nC43sf7QrgbZ1QGBwC8NAoLECgYBNgNoKCaVQz2z38rs6pdkS8lUPdgL6APrZUTcY\n/oukHftzOl6XERw1dt7teA9XJmEy+9ezoob133yxapHD3HEGRuELg7gYIHLI/7VY\nshtGM1SfcOBRwY5aNEh7Bohrnh7hOGao6C7QnBHCsDUyz9huNpJ+3vH5eGcji7M2\nJXM0LwKBgQCIA/4ku6AZ27JTxTbeN94LG9+7U3Gy45fE343uZNpubseJL4dTMq/n\nmy1CssCR4D56HFq6GcRDJkpgN2Fsfg9lY8ibMb7vdW8X1HB/jwypVwMzXmMAcsRD\nQnrVGD15EEaezEDCXfINEjj1HgeEDWtVESRYJA4RqMLjPbtyfylcow==\n-----END RSA PRIVATE KEY-----\n"
```

Comment:
Pri pisani dokumentacie som si uvedomil, ze to musi byt zasifrovane, kedze obsahuje private key -> identitu klienta.